internal/adapters/scheduler_storage/pg/pg.go

```diff
@@ -52,7 +52,16 @@ SELECT
 	v.rolling_update_status, v.rollback_version
 FROM schedulers s join scheduler_versions v
 	ON s.name=v.name AND v.version=s.version
-WHERE s.name = ?`
+WHERE s.name = ?
+order by created_at desc`
+	queryGetSchedulerByVersion = `
+SELECT
```

Comment:
@caiordjesus queryGetScheduler and queryGetSchedulerByVersion are identical queries, in but queryGetSchedulerByVersion hasn't order by by created_at. I think it was because the query QueryGetSchedulerByVersion should returns one row, right? if not i think you can use order by like in first query.